src/resources/vue/components/model.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+    <div class="bg-modal" @click="toggleModal" v-if="showModal">
+        <div v-if="showModal" class="modal" @click.stop>
+            <svg class="close-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" style="transform: rotate(45deg);">
+                <path d="M11 11V5H13V11H19V13H13V19H11V13H5V11H11Z"></path>
+            </svg>
+            <!-- <p class="modal-content">This is some content in the modal.</p> -->
+            <slot></slot>
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    data() {
+        return {
+            showModal: true
+        };
+    },
+    methods: {
+        toggleModal() {
+            this.showModal = !this.showModal;
+        }
+    }
+};
+</script>
+
+<style scoped>
```

Comment:
Better formatting please :)

Reply:
install the prettier extension and press ctrl+alt+f to format using prettier

Reply:
fixed!